src/aleph/http/client.clj

```diff
@@ -367,7 +386,11 @@
   (fn [^ChannelPipeline pipeline]
     (let [handler (if raw-stream?
                     (raw-client-handler response-stream response-buffer-size)
-                    (client-handler response-stream response-buffer-size))]
+                    (client-handler response-stream response-buffer-size))
+          logger (when (some? log-activity)
+                   (LoggingHandler.
+                    "aleph-client"
+                    ^LogLevel (coerce-log-level log-activity)))]
```

Comment:
Oh, and to avoid the type hint here it could be included directly in the `coerce-log-level` function signature instead:

```clojure
(defn ^LogLevel coerce-log-level [level]
...
)
```

I know, I'm usually picky about such little details ;-)